psalm.xml.dist

```diff
@@ -4,6 +4,7 @@
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xmlns="https://getpsalm.org/schema/config"
     xsi:schemaLocation="https://getpsalm.org/schema/config vendor/vimeo/psalm/config.xsd"
+    errorBaseline="psalm-baseline.xml"
```

Comment:
Hmm, we were at zero baseline: kinda sad to be back to this 😬

Still, patch moves us forward 👍